pkg/asset/machines/cluster_k8s_io.go

```diff
@@ -0,0 +1,73 @@
+package machines
+
+import (
+	"bytes"
+	"text/template"
+
+	"github.com/openshift/installer/pkg/asset"
+	"github.com/openshift/installer/pkg/asset/installconfig"
+	"github.com/openshift/installer/pkg/types"
+)
+
+// ClusterK8sIO generates the `Cluster.cluster.k8s.io/v1alpha1` object.
+type ClusterK8sIO struct {
+	Raw []byte
+}
+
+var _ asset.Asset = (*ClusterK8sIO)(nil)
+
+// Name returns a human friendly name for the ClusterK8sIO Asset.
+func (c *ClusterK8sIO) Name() string {
+	return "Cluster.cluster.k8s.io/v1alpha1"
+}
+
+// Dependencies returns all of the dependencies directly needed by the
+// ClusterK8sIO asset
+func (c *ClusterK8sIO) Dependencies() []asset.Asset {
+	return []asset.Asset{
+		&installconfig.InstallConfig{},
+	}
+}
+
+// Generate generates the Worker asset.
+func (c *ClusterK8sIO) Generate(dependencies asset.Parents) error {
+	installconfig := &installconfig.InstallConfig{}
+	dependencies.Get(installconfig)
+
+	c.Raw = clusterK8sIO(installconfig.Config)
+	return nil
+}
+
+var clusterK8sIOTmpl = template.Must(template.New("cluster").Parse(`
+apiVersion: "cluster.k8s.io/v1alpha1"
+kind: Cluster
+metadata:
+  name: {{.Name}}
+  namespace: openshift-cluster-api
+spec:
+  clusterNetwork:
+    services:
+      cidrBlocks:
+      - {{.ServiceCIDR}}
+    pods:
+      cidrBlocks:
+      - {{.PodCIDR}}
+    serviceDomain: unused
```

Comment:
We've had this since openshift/machine-api-operator@04253cb9, but I'm not clear on why we aren't using our base domain or some such.  @enxebre, do you remember why you chose this value?